src/NzbDrone.Core/Statistics/HistoryStatistics.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Statistics
+{
+    public class HistoryStatistics
```

Comment:
The original thought was that this particular statistic comes from the History, I thought that I might find other useful sources to analyse.